src/dataset.py

```diff
@@ -54,9 +51,14 @@ def get_affine_transform(
     return trans
 
 
-def affine_transform(pt, t):
-    new_pt = np.array([pt[0], pt[1], 1.], dtype=np.float32).T
-    new_pt = np.dot(t, new_pt)
+def affine_transform(point: np.array, transform_matrix: np.array) -> np.array:
+    """
+    :param point: np.ndarray with shape (2, )
```

Comment:
we use [Google style docstring](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html)